tests/apps/actordotnet/CarActor.cs

```diff
@@ -0,0 +1,36 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace DaprDemoActor
+{
+    using Dapr.Actors.Runtime;
+    using System.Threading.Tasks;
+    using System.Text.Json;
+
+    [Actor(TypeName = "DotNetCarActor")]
+    public class CarActor : Actor, ICarActor
+    {
+        private int count = 0;
+
+        public CarActor(ActorHost host) : base(host)
+        {
+        }
+
+        public Task<int> IncrementAndGetAsync(int delta) {
+            count += delta;
+            return Task.FromResult(count);
+        }
+
+        public Task<string> CarToJSONAsync(Car car) {
```

Comment:
This isn't really a representative test of how someone would use actors (the `Task<string>` part). This should return `Task<Car>` and the framework should handle serialization.

Reply:
Same comment as well for `CarFromJSONAsync`

Reply:
This is by design to test string input/output. Object input and output.

Reply:
Just to clarify. E2E test apps do not necessarily represent a real world scenario: samples do that. See the other apps we have for E2E.

Reply:
I guess I don't quite get why strings are significant - does it have special handling at the runtime layer? I would expect this to produce/accept a JSON-encoded string, which is one of the things most likely to be compatible across different languages/runtimes.

Reply:
I don't want to micro-focus on this since it's still a reasonable test.

Reply:
@rynowak Strings can be handled differently in the serialization layer. There was an issue with Java where strings where not being serialized correctly because it was handled in a special way. Strings, when encoded into JSON, get quoted, while the Java SDK was passing them as-is. This issue is not there anymore but this test will make sure it stays that way.